### DISCUSSION TITLE
Need help with config of whoogle CT

### DISCUSSION THREAD

**Discussion by comfuzio** [tteck/Proxmox#954] [Help Needed]
Hi and thanks for the amazing tools!

I have installed the whoogle CT but I am not familiar with it and it's configuration, where are it's config files located at?
Because all searches are in German first and I would like to change it :)

Thanks :)

Kind regards,
George

**Comment by tteck** [comment]
Have you tried the configuration on the main page?

![Screenshot 2023-01-19 1 36 12 AM](https://user-images.githubusercontent.com/21342670/213372716-1647eb39-825a-4561-82cb-43c2cb1f8912.png)

**Reply by comfuzio** [reply]
Yes, remains the same no matter what. 
I assume it has a configuration in the container that i can change that because i see in whoogle's github guide they have some options you can set but it doesn't say for this script which i assume it is the python version right?

Kind regards, George

**Comment by comfuzio** [comment]
@tteck can you try at https://whoogle.gr if works for you? If you switch settings, does it save them for you?
Maybe it is an issue on my end but I have tried it on Edge/Chrome/Firefox in both Windows 11 and Manjaro linux and it behaved the same...I am not an expert, rather I am a noob at it and I do not want to write things that are wrong, I saw the whoogle's github and they have a config file if you do it via docker or docker compose where you can specify a few settings. I have tried to look for that config file in this CT but I wasn't able to locate it.
Maybe if we had some documentation I could locate it and change the defaults to Greek or English rather than German.
The "important" thing is those settings would be by default for all users :)

Thanks!

**Comment by tteck** [comment]
Everything is working as expected. In that configuration screen (self hosted), if you set everything as you want, then hit "save as" name it "english" a file is written to `/usr/local/lib/python3.9/dist-packages/app/static/config/` named `english.conf` Then you can "load" it, or load another `.conf` with different settings.

**Comment by comfuzio** [comment]
![image](https://user-images.githubusercontent.com/30821243/213876724-208f2314-02d2-4a1d-9ff0-c6cf4db73a27.png)
for me it doesn't work as expected, i set all Greek and apply, it remains as you see in the pic and all search results come back in German...
I will ask in whoogle's as well.
Thanks @tteck

**Comment by comfuzio** [comment]
hi, in the end i moved it to docker where i can do configs without issues and helped in translating the project.
i would rather to have it in CT but with no config ability and all results coming back in German or from German sites, was a no no
